src/services/ngx-uploader.ts

```diff
@@ -30,13 +30,18 @@ export class NgUploaderService {
 
   uploadFile(file: any): void {
     let xhr = new XMLHttpRequest();
-    let form = new FormData();
+    let payload: any;
```

Comment:
No, I renamed form to payload, and it's set again just below, when multipart is on. This is because, when `multipart==false`, var form|payload isn't of type `FormData`, but set directly to `file` (that is received as `any`). That's why I just declare but don't set it here, and set it in both `if` cases below.